### PR TITLE
fix timeout and theme var assignments

### PR DESCRIPTION
**Pull request by xavluiz**
assigning the ping timeout to undefined to prevent an undefined error and upgrading the mui theme api based on the material update

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -33,9 +33,6 @@ export function isTelemetryOn() {
 }
 
 export function deactivate(ctx: ExtensionContext) {
-  // Process this window's keystroke data since the window has become unfocused/deactivated
-  commands.executeCommand("codetime.processKeystrokeData");
```
Comment (mmclead) [inline src/extension.ts:37]: what's the reason for this change?

**Review by mmclead — APPROVED** [review]

File: src/extension.ts (hunk 1 above)
Comment (xavluiz) [reply src/extension.ts:37]: oh, this gets called before the command has been registered. I don't think we need it in the deactivate anyway. What do you think. It was was also reported by a user that we received the command not found exception. So I'm not exactly sure how the deactivate gets called before the commands are registered but somehow it can.

Comment (mmclead) [reply src/extension.ts:37]: got it.   all good.    that makes sense.
